libs/parallelism/futures/include/hpx/futures/traits/detail/future_await_traits.hpp

```diff
@@ -343,6 +346,8 @@ namespace std {
                     this->base_type::set_value(std::forward<U>(value));
                 }
 
+                void unhandled_exception() const noexcept {}
+
```

Comment:
Shouldn't this be added to the specialization for `hpx::lcos::future<void>` above and `hpx::lcos::shared_future<void>` below as well?

Reply:
Definitely. The tests also don't compile yet. This is just playing around. If you happen to know what's missing feel free to push here (though I know you might not have the time). I need significantly more time to understand the implementation as I've never looked at it properly before.